### PR TITLE
NFV DT's

### PR DESCRIPTION
**Pull request by eshulman2**
- created DT descriptions for some of our jobs, still need to discuss different HW

### CODE REVIEW

File: dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Stages
+
+All stages must be executed in the order listed below.  Everything is required unless otherwise indicated.
+
+1. [Install dependencies for the OpenStack K8S operators](stage1)
+2. [Install the OpenStack K8S operators](stage2)
+3. [Configuring networking on the OCP nodes](stage3)
+4. [Configure and deploy the control plane](stage4)
+5. [Configure and deploy the initial data plane to prepare for CephHCI installation](stage5)
```
Comment (agarcia64) [inline dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp.md:47]: Are you sure that ceph is needed in this?

File: dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Stages
+
+All stages must be executed in the order listed below.  Everything is required unless otherwise indicated.
+
+1. [Install dependencies for the OpenStack K8S operators](stage1)
+2. [Install the OpenStack K8S operators](stage2)
+3. [Configuring networking on the OCP nodes](stage3)
+4. [Configure and deploy the control plane](stage4)
+5. [Configure and deploy the initial data plane to prepare for CephHCI installation](stage5)
```
Comment (agarcia64) [inline dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid.md:49]: Same here, is ceph needed?

File: dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp.md (hunk 1 above)
Comment (eshulman2) [reply dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-hybrid-mq-balance-tcp-lacp-igmp.md:47]: Eliad just asked to ignore this part for now, so I would say no but it wont hurt

File: dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid.md (hunk 2 above)
Comment (eshulman2) [reply dt/NFV/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid/DFG-nfv-18.0-3cont-2comp-ipv4-geneve-ovn-dpdk-sriov-ctlplane-dataplane-bonding-rt-hybrid.md:49]: Eliad just asked to ignore this part for now, so I would say no but it wont hurt

**Pull request closed without merging**